pmd-apex/src/main/resources/rulesets/apex/quickstart.xml

```diff
@@ -0,0 +1,444 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ruleset xmlns="http://pmd.sourceforge.net/ruleset/2.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" name="Default ruleset used by the CodeClimate Engine for Salesforce.com Apex" xsi:schemaLocation="http://pmd.sourceforge.net/ruleset/2.0.0 https://pmd.sourceforge.io/ruleset_2_0_0.xsd">
+   <description>Default ruleset used by the Code Climate Engine for Salesforce.com Apex</description>
```

Comment:
We need to adjust the ruleset name and description. In java we use as name just "quickstart" and as description "Quickstart configuration of PMD for Apex. Includes rules that are most likely to apply everywhere".